extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js

```diff
@@ -574,6 +574,18 @@ describes.realWin('amp-ad-network-doubleclick-impl', realWinConfig, env => {
         });
       });
     });
+    it('should have google_preview parameter', () => {
```

Comment:
can we add an sra test in test-doubleclick-sra too? just a simple addition to an existing test that that the gct param gets added

Reply:
Done